articles/20240822_v0.24.0_news.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Daytona v0.24.0"
```

Comment:
Please invest more effort to double check everything, you are mixing version numbers
Also please read the issue, this needs to be a fluently written article.